### PR TITLE
Adding percentile metric value properties camel case suppression

### PR DESCRIPTION
**Pull request by dmakwana**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cosmos-db/resource-manager/readme.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ directive:
     reason: The DatabaseAccount request and response resources differ so the DatabaseAccountCreateUpdateParameters must only have a PUT operation
   - suppress: DefinitionsPropertiesNamesCamelCase
     from: cosmos-db.json
-    where: $.definitions.MetricValue.properties._count
+    where: $.definitions.MetricValue.properties._count, $.definitions.PercentileMetricValue
```
Comment (dsgouda) [inline specification/cosmos-db/resource-manager/readme.md]: Has this suppression been approved by the review committee?

**Comment by lmazuel** [comment]
@dmakwana Seems the syntax is not correct, see the Python job for more details.
@dsgouda There is an email thread in process, but let's work on this PR to make it green (it's 99% likely that this will be accepted for what I see so far). Python job failing is relevant, and needs to be fixed until it's not green.

**Comment by dmakwana** [comment]
@lmazuel Fixed the syntax errors.

@ravbhatnagar

**Comment by lmazuel** [comment]
@dmakwana I looked at the generated code, you need to add `"x-ms-parameter-location": "method",` to `sourceRegionParameter`, `targetRegionParameter` and `partitionKeyRangeIdParameter` too.

**Review by lmazuel — CHANGES_REQUESTED** [review]
@dmakwana I looked at the generated code, you need to add "x-ms-parameter-location": "method", to sourceRegionParameter, targetRegionParameter and partitionKeyRangeIdParameter too.

**Review by lmazuel — APPROVED** [review]

**Comment by lmazuel** [comment]
Got the Go from @salameer

**Pull request merged**